<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-NVR46R0" name="Default test" tests="7" failures="1" timestamp="11 Mar 2022 10:05:14 GMT" time="77.538" errors="0">
  <testcase name="select_customer_name" time="1.696" classname="IBM.automation_1.retail_1"/>
  <testcase name="select_mail_option" time="2.321" classname="IBM.automation_1.retail_1"/>
  <testcase name="select_model" time="1.858" classname="IBM.automation_1.retail_1"/>
  <testcase name="select_order_id" time="1.754" classname="IBM.automation_1.retail_1"/>
  <testcase name="select_product" time="1.172" classname="IBM.automation_1.retail_1"/>
  <testcase name="select_return_id" time="1.168" classname="IBM.automation_1.retail_1"/>
  <testcase name="select_total" time="2.477" classname="IBM.automation_1.retail_1">
    <failure type="junit.framework.ComparisonFailure" message="expected:&amp;lt;Rs.85&amp;gt; but was:&amp;lt;&amp;gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<Rs.85> but was:<>
	at junit.framework.Assert.assertEquals(Assert.java:81)
	at junit.framework.Assert.assertEquals(Assert.java:87)
	at IBM.automation_1.retail_1.select_total(retail_1.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:663)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:849)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- select_total -->
</testsuite> <!-- Default test -->
